src/app/pages/MediaArticlePage/PagePromoSections/LatestMediaSection/index.styles.tsx

```diff
@@ -9,6 +9,9 @@ const styles = {
       [mq.GROUP_3_ONLY]: {
         margin: '0 1rem',
       },
+      [mq.GROUP_4_MIN_WIDTH]: {
+        margin: '0rem',
```

Comment:
Not sure you need 'rem'? 0 is 0 whatever the unit?

Reply:
good spot! Have fixed